src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,432 @@
+---
+title: Monitor your Flutter app
+tags:
+    - Flutter
+    - Mobile monitoring
+    - New Relic Mobile Flutter
+    - Get started
+metaDescription: "New Relic's Flutter agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterOverview from 'images/mobile_screenshot-full_flutter-overview.png'
+
+Monitor your Flutter application with our New Relic Flutter agent. Capture Dart errors, track specific network requests, and use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
+
+<img
+  title="mobile UI for flutter "
+  alt="mobile UI for flutter "
+  src={mobileFlutterOverview}
+/>
+
+<figcaption>
+  
+</figcaption>
```

Comment:
@ndesai-newrelic Can you provide a screenshot of sample Flutter app and some populated data in NR?